### PR TITLE
Adding ring-handler pointing all requests at single page

### PR DESCRIPTION
**Pull request by smogg**

### CODE REVIEW

File: src/desdemona/ui/server.clj [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    [:body
+     [:div#app]
+     (include-js "/js/main.js")]]))
+
+(defn handler [request]
+  {:status 200
+   :headers {"Content-Type" "text/html"}
+   :body page})
+
+(def spa-server
```
Comment (lvh) [inline src/desdemona/ui/server.clj]: What's a spa server?

**Comment by lvh** [comment]
This looks fine to me as long as you add test coverage :) Fortunately ring handler functions are easy to test: just call it and see what happens...

File: src/desdemona/ui/server.clj (hunk 1 above)
Comment (smogg) [inline src/desdemona/ui/server.clj]: SPA = single page app. Probably not the best name... How about just "server"?

Comment (lvh) [inline src/desdemona/ui/server.clj]: Derp. I was thrown off because SPA used to be the old name for RMS :D Just server is fine.

**Comment by lvh** [comment]
The reduced coverage is a bug; the macros are used correctly; they just expand to two branches, one of which isn't covered.

File: test/desdemona/ui/server_test.clj [hunk 2]
```diff
@@ -0,0 +1,16 @@
+(ns desdemona.ui.server-test
+  (:require [clojure.test :refer [deftest is]]
+            [desdemona.ui.server :as s]
+            [ring.mock.request :refer [request]]))
+
+(deftest page-test
```
Comment (lvh) [inline test/desdemona/ui/server_test.clj]: It'd be nice if this was a test resource that got slurped, but that's not really necessary :)

**Pull request merged**